pagopa/io-app
# feat: [IOCOM-873] IUN code on SEND message details with new DS

## Short description
This PR adds the "Notification Info" section to the SEND message details, with the new DS.
|Message Info|
|-|
|![Simulator Screenshot - iPhone 15 - 2024-03-27 at 17 28 19](https://github.com/pagopa/io-app/assets/5150343/2e9adf24-9b03-4db4-97fd-217cd23a02c6)|

## List of changes proposed in this pull request
- New `MessageInfo` component that shows the IUN code (with the copy-to-clipboard button)
- `MessageInfo` added to `MessageDetails`

## How to test
Using the io-dev-api-server, generate some SEND messages. Open such a message with the new DS enabled and check that the IUN code is properly displayed and it can be copied into the clipboard.
